Gems/Terrain/Code/Source/TerrainRenderer/TerrainMeshManager.cpp

```diff
@@ -125,20 +150,33 @@ namespace Terrain
 
         for (uint32_t i = 0; i < m_sectorStack.size(); ++i)
         {
-            const float maxDistance = m_config.m_firstLodDistance * aznumeric_cast<float>(1 << i);
-            const float gridMeters = (GridSize * m_sampleSpacing) * (1 << i);
-            const int32_t startCoordX = aznumeric_cast<int32_t>(AZStd::floorf((newPosition.GetX() - maxDistance) / gridMeters));
-            const int32_t startCoordY = aznumeric_cast<int32_t>(AZStd::floorf((newPosition.GetY() - maxDistance) / gridMeters));
-
-            // If the start coord for the stack is different, then some of the sectors will need to be updated.
             StackData& stackData = m_sectorStack.at(i);
-            if (stackData.m_startCoordX != startCoordX || stackData.m_startCoordY != startCoordY)
+
+            auto [newStartCoordX, newStartCoordY] = [&]()
+            {
+                const float maxDistance = m_config.m_firstLodDistance * aznumeric_cast<float>(1 << i);
+                const float gridMeters = (GridSize * m_sampleSpacing) * (1 << i);
+                const int32_t startCoordX = aznumeric_cast<int32_t>(AZStd::floorf((newPosition.GetX() - maxDistance) / gridMeters));
+                const int32_t startCoordY = aznumeric_cast<int32_t>(AZStd::floorf((newPosition.GetY() - maxDistance) / gridMeters));
+
+                // If the start coord for the stack is different, then some of the sectors will need to be updated.
+                // There's 1 sector of wiggle room, so make sure we've moving the lod's start coord by as little as possible.
+
+                const int32_t newStartCoordX = startCoordX > stackData.m_startCoordX + 1 ? startCoordX - 1 :
+                    (startCoordX < stackData.m_startCoordX ? startCoordX : stackData.m_startCoordX);
+                const int32_t newStartCoordY = startCoordY > stackData.m_startCoordY + 1 ? startCoordY - 1 :
```

Comment:
It's something I considered, but I think it's best to hold off for now because for now it wouldn't really help much with how things currently work.  I think it may make sense to do in the future if we change the sector building to be fully async instead of just parallelized - then we could give sectors a few frames to build as long as the camera wasn't moving too fast.